doozer

```diff
@@ -1552,6 +1552,7 @@ def release_mirror(runtime, src_dest, image_stream, is_base, pattern):
     src_dest_items = []
     missing_source_items = []
     invalid_name_items = []
+    blacklisted_items = []
```

Comment:
This doesn't get populated anywhere that I see.... what am I missing? what's the point?